doc/docs_es.md

```diff
@@ -0,0 +1,899 @@
+array (`cap`), su longitud inicial (`len`)
+y el elemento por defecto (`init`):
+
+```v
+arr := []int{len: 5, init: -1}
+// `[-1, -1, -1, -1, -1]`
+```
+
+El ajuste de la capacidad mejora el rendimiento de las inserciones
+ya que reduce el número de reasignaciones necesarias:
+
+```v
+mut numbers := []int{cap: 1000}
+println(numbers.len) // 0
+// Ahora al añadir elementos no se reasignan
+for i in 0 .. 1000 {
+	numbers << i
+}
+```
+
+Nota: El código anterior utiliza una sentencia [range `for`](#range-for).
+
+#### Métodos de matrices
+
+Todas las matrices pueden imprimirse fácilmente con `println(arr)` y convertirse en una cadena
+con `s := arr.str()`.
+
+La copia de los datos de la matriz se realiza con `.clone()`:
+
+```v
+nums := [1, 2, 3]
+nums_copy := nums.clone()
+```
+
+Las matrices pueden ser filtradas y mapeadas eficientemente con los métodos `.filter()` y
+`.map()`:
+
+```v
+nums := [1, 2, 3, 4, 5, 6]
+even := nums.filter(it % 2 == 0)
+println(even) // [2, 4, 6]
+// filter puede aceptar funciones anónimas
+even_fn := nums.filter(fn (x int) bool {
+	return x % 2 == 0
+})
+println(even_fn)
+words := ['hello', 'world']
+upper := words.map(it.to_upper())
+println(upper) // ['HELLO', 'WORLD']
+// map también puede aceptar funciones anónimas
+upper_fn := words.map(fn (w string) string {
+	return w.to_upper()
+})
+println(upper_fn) // ['HELLO', 'WORLD']
+```
+
+`it` es una variable incorporada que se refiere al elemento que
+se está procesando actualmente en los métodos filter/map.
+
+#### Matriz multidimensional
+
+Las matrices pueden tener más de una dimensión.
+
+Ejemplo de matriz de 2 dimensiones:
+
+```v
+mut a := [][]int{len: 2, init: []int{len: 3}}
+a[0][1] = 2
+println(a) // [[0, 2, 0], [0, 0, 0]]
+```
+
+Ejemplo de matriz de 3 dimensiones:
+
+```v
+mut a := [][][]int{len: 2, init: [][]int{len: 3, init: []int{len: 2}}}
+a[0][1][1] = 2
+println(a) // [[[0, 0], [0, 2], [0, 0]], [[0, 0], [0, 0], [0, 0]]]
+```
+
+#### Ordenación de matrices
+
+Ordenar matrices de todo tipo es muy sencillo e intuitivo. Las variables especiales `a` y `b`
+se utilizan cuando se proporciona una condición de ordenación personalizada.
+
+```v
+mut numbers := [1, 3, 2]
+numbers.sort() // 1, 2, 3
+numbers.sort(a > b) // 3, 2, 1
+```
+
+```v nofmt
+struct User {
+	age  int
+	name string
+}
+
+mut users := [User{21, 'Bob'}, User{20, 'Zarkon'}, User{25, 'Alice'}]
+users.sort(a.age < b.age) // ordenación por el campo de entero User.age
+users.sort(a.name > b.name) // ordenación inversa por el campo de cadena User.name
+```
+
+#### Rebanadas de la matriz (Array slices)
+
+Las rebanadas son matrices parciales. Representan cada elemento entre dos índices
+separados por un operador .. El índice del lado derecho debe ser mayor o igual
+al índice del lado izquierdo.
+
+Si un índice del lado derecho está ausente, se asume que es la
+longitud de la matriz. Si no hay un índice
+índice del lado izquierdo está ausente, se asume que es 0.
```

Comment:
proofread this part - you have two different, incomplete sentences here :)

Reply:
resolved